tools/container/toolkit/toolkit.go

```diff
@@ -450,6 +451,12 @@ func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContai
 		"nvidia-container-runtime-hook.path":                nvidaContainerRuntimeHookPath,
 		"nvidia-container-runtime-hook.skip-mode-detection": opts.ContainerRuntimeHookSkipModeDetection,
 	}
+
+	toolkitRuntimeList := opts.ContainerRuntimeRuntimes.Value()
+	if len(toolkitRuntimeList) > 0 {
+		configValues["nvidia-container-runtime.runtimes"] = toolkitRuntimeList
+	}
+
```

Comment:
Below we already set:
```
"nvidia-container-runtime.runtimes": opts.ContainerRuntimeRuntimes,
```
meaning that this change should not be required. If that isn't working as expected, then we need to address that.

```suggestion
```

Reply:
The `optionalConfigValues` are only set if the CLI flag is set. See [here](https://github.com/NVIDIA/nvidia-container-toolkit/blob/3e6c2b7d18a8f5faa9a2258bb38076f7c6da7dad/tools/container/toolkit/toolkit.go#L475)

Reply:
You're right. Let's keep it as is then.